<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUp</class>
 <widget class="QDialog" name="SignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>300</height>
      </size>
     </property>
     <property name="title">
      <string>SignUp</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0" colspan="3">
       <widget class="QLineEdit" name="userName"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>138</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Already Have an Account?</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QLineEdit" name="firstName"/>
      </item>
      <item row="7" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QLineEdit" name="email"/>
      </item>
      <item row="6" column="0" colspan="3">
       <widget class="QPushButton" name="signup_button">
        <property name="text">
         <string>SignUp</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="3">
       <widget class="QLineEdit" name="confirmPassword"/>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QLineEdit" name="password"/>
      </item>
      <item row="7" column="1">
       <widget class="QPushButton" name="signin_button">
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QLineEdit" name="secondName"/>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="signup_text">
        <property name="maximumSize">
         <size>
          <width>127</width>
          <height>10</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: red;
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt; font-weight:600;&quot;&gt;NotePad&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
